photon-targeting/src/generated/main/java/org/photonvision/struct/PhotonTrackedTargetSerde.java

```diff
@@ -90,35 +91,35 @@ public PhotonTrackedTarget unpack(Packet packet) {
 
         // yaw is of intrinsic type float64
         ret.yaw = packet.decodeDouble();
-
+    
         // pitch is of intrinsic type float64
         ret.pitch = packet.decodeDouble();
-
+    
         // area is of intrinsic type float64
         ret.area = packet.decodeDouble();
-
+    
```

Comment:
Random whitespace. Can we get the template to get rid of these?

Reply:
Doneski